test/jit/test_profiler.py

```diff
@@ -0,0 +1,25 @@
+import os
+import sys
+
+import torch
+
+# Make the helper files in test/ importable
+pytorch_test_dir = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+sys.path.append(pytorch_test_dir)
+from torch.testing._internal.jit_utils import JitTestCase
+
+if __name__ == '__main__':
+    raise RuntimeError("This test file is not meant to be run directly, use:\n\n"
+                       "\tpython test/test_jit.py TESTNAME\n\n"
+                       "instead.")
+
+class TestProfiler(JitTestCase):
```

Comment:
Is it used anywhere?

Reply:
it's imported in test_jit.py, test cases are purposefully empty

Reply:
What's the purpose of empty tests?

Reply:
so the next PR that adds tests here has a smaller set of changes